js/cell.js

```diff
@@ -22,39 +23,20 @@ define([
         r: "r",
     };
 
-    var style = document.createElement("style");
-    style.innerHTML = css.replace(
-        /url\((?!data:)/g,
-        "url(" + utils.URLs.root + "static/"
-    );
-    var fs = document.getElementsByTagName("script")[0];
-    fs.parentNode.insertBefore(style, fs);
+    domReady(function () {
```

Comment:
I plan to migrate to webpack. It gives support for es modules, source maps, minification, and transpiling. It appears minification is currently broken on master (not sure how long it has been like this...). `requirejs` was last updated 4 years ago...

Reply:
Makes sense and your work is greatly appreciated! When I picked up, everything was in a couple of files that were half requirejs and half not. I've tried to break it into pieces and use requirejs for everything except for the top level loading, so that old webpages were still working. Didn't consider any alternatives and didn't look at the activity either.